modules/exploits/windows/local/docker_credential_wincred.rb

```diff
@@ -0,0 +1,104 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ManualRanking
+
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+  include Post::Windows::Priv
+  include Post::Windows::Runas
+
+  def initialize(info = {})
+    super(
+      update_info(info,
+        'Name'          => 'Docker-Credential-Wincred.exe priviledge  escallation',
```

Comment:
There's some extra whitespace and a typo here I think :eyes:

```suggestion
        'Name'          => 'Docker-Credential-Wincred.exe priviledge escalation',
```